content-enricher/content-enricher.bal

```diff
@@ -0,0 +1,36 @@
+import ballerina/http;
```

Comment:
Don't we need to add the license header?

Reply:
Let's not do per file. We have a license for the repo.